README.md

```diff
@@ -210,3 +211,261 @@ developer console](https://code.google.com/apis/console/).
 ### Format of the payload
 
 This is the same format as [the reminder from the reminders API](#get-remindersid).
+
+## API users
+
+This API manages users.
+
+### POST `/users`
+
+This creates a new user.
+
+#### Input
+
+All properties must be present. There is no default value. For example:
+```json
+{
+  "forename": "Sherlock",
+  "email": "Sherlock.Holmes@baker-street.co.uk",
+  "password": "MoriartyShallReturn"
+}
+```
+
+Note that the password is stored in database using a modern hash algorithm
+(Argon2 currently).
+
+Currently the user is created as-is. In the future we'll implement a 2-step
+creation mechanism where the email would be used.
+
+#### Output
+* 201 if request succeeded, with `location` header indicating the new resource URL.
```

Comment:
How can we easily get the content of the location header on the app?
For the sake of consistency with the reminders API it should also return the user object including its newly created ID.